src/presentation/styles/framerMotionComponents/HoverButton.js

```diff
@@ -30,6 +30,21 @@ export class HoverButtonClick extends Component {
     }
 }
 
+export class HoverButtonLink extends Component {
+    render() {
+        return (
+                <motion.div whileHover={{ scale: 1.2 }}>
+                    <Link to={this.props.linky}>
+                    <Button onClick={this.props.clickFunction} style={{backgroundColor: '#3c6cd7', transition: "ease background-color 250ms", hover: {backgroundColor: '#3460c4'}}} >
+                        {this.props.text}
+                    </Button>
+                    </Link>
+                </motion.div>
+
+        )
+    }
+}
+
```

Comment:
For a component this simple, I believe it is best to use a React Hook in a function, rather than a React class.